### DISCUSSION TITLE
Buttons guidance: change examples to conform to Carbon capitalization guidelines

### DISCUSSION THREAD

**Discussion by ericradz** [carbon-design-system/carbon#17377] [Release Feedback]
Can the guidance for Buttons be updated so that all examples that include a multi-word label use sentence-style capitalization? For example, https://react.carbondesignsystem.com/?path=/docs/components-button--overview#overview contains examples that use `Tertiary Danger Button`, `Ghost Danger Button`, `Danger Tertiary`, and `Danger Ghost`.  As per the [Capitalization](https://carbondesignsystem.com/guidelines/content/writing-style#capitalization) guideline, these should all use sentence-style caps.

**Comment by adamalston** [comment]
https://github.com/carbon-design-system/carbon/issues/18673

**Reply by adamalston** [reply]
This discussion should be addressed now that the linked issue has been addressed.
